_docs/sysadmin/system_customization.md

```diff
@@ -166,6 +168,32 @@ However, this should be only applied to the `php.ini` running the web server and
 
 <small>[Back To Table of Contents](#table-of-contents)</small>
 
+### Increasing the max number of files that can be uploaded at once
+
+By default PHP restricts uploading 20 files at a time. To change this limit, edit:
+
+```
+/etc/php/7.2/fpm/php.ini
+```
+
+and modify the variable:
+
+```
+upload_max_filesize
+```
+
+Then restart PHP and apache
```

Comment:
Remove the restart of apache (as not needed).

Change the restart of PHP-FPM to a reload: `systemctl reload php7.2-fpm`